Karamelfile

```diff
@@ -9,6 +9,7 @@ dependencies:
       - hops::dn
   - recipe: hadoop_spark::yarn
     global:
+      - hadoop_spark::install
```

Comment:
@SirOibaf isn't this redundant? I thought install recipes always run before the others.

Reply:
Yes, Karamel runs all the install recipe before any other recipe of the same cookbook is run.

Reply:
Fixed